src/script/util/util.ts

```diff
@@ -398,3 +398,6 @@ export const getSelectionPosition = (element: HTMLTextAreaElement, currentMentio
 
   return {newEnd, newStart};
 };
+
+// temporary hack that disables mls for old 'broken' desktop clients, see https://github.com/wireapp/wire-desktop/pull/6094
+export const isMLSDisabledOnElectron = Runtime.isDesktopApp() && !window.systemCrypto;
```

Comment:
I think this condition will disable mls for webapp not in electron

```suggestion
export const supportsMLS = !Runtime.isDesktopApp() || window.systemCrypto;
```